packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1979,9 +1979,6 @@ export default mixins(
 				// Automatically deselect all nodes and select the current one and also active
 				// current node
 				this.deselectAllNodes();
-				setTimeout(() => {
-					this.nodeSelectedByName(newNodeData.name, true);
-				});
```

Comment:
this does more than just not open NDV. this also selects the node. we should keep the behaviour that new node is selected.